heimdall2/values.yaml

```diff
@@ -91,18 +130,29 @@ heimdall:
                 name: heimdall
                 port:
                   number: 3000
+    annotations:
+    #   traefik.ingress.kubernetes.io/router.entrypoints: websecure
+    #   traefik.ingress.kubernetes.io/router.tls: "true"
+    #   traefik.ingress.kubernetes.io/service.sticky.cookie: "true"
+      ## Change this annotation to false if using an ingress that is not the default for the cluster - you will probably also want to set `className`.
+      ingressclass.kubernetes.io/is-default-class: "true"
+    # className: ingress class name
+    # tls:
+    #   - name: nginx
+    #     secretName: ingress-secret
+    #     hosts:
```

Comment:
Should we have a hosts.host value here if we have one on line 124? Would those two values always be the same or no?

Reply:
I don't know.  @wdower can you provide any input?